tests/test_cases/test_iteration_vhdl/Makefile

```diff
@@ -28,10 +28,10 @@
 include ../../designs/viterbi_decoder_axi4s/Makefile
 MODULE = test_iteration
 
-ifeq ($(SIM),$(filter $(SIM), ius xcelium))
+ifneq ($(filter $(SIM),ius xcelium),)
     SIM_ARGS += -notimezeroasrtmsg
 endif
 
-ifeq ($(SIM),$(filter $(SIM), ghdl))
+ifeq ($(SIM),ghdl)
```

Comment:
For consistency can we use the same spelling as above here?

Reply:
I see there are more places like this. Should I change everywhere?

Reply:
I suppose we already use this format elsewhere, so probably not important.

Reply:
I the meantime I corrected every else (also for Questa). Is this OK for you?

Reply:
Either is fine